test/run_tests.py

```diff
@@ -101,6 +101,7 @@ def signal_handler(sgnl, frame):
     result = test_console.test_lsmagic()
 
 if result:
+    test_console.test_lsmagic()
```

Comment:
the script should fail if the tests fail.
there should be only one call to test_lsmagic.
rename test_lsmagic to test_ipython.